docs/articles/baseplatform/repo-manifests.rst

```diff
@@ -1,6 +1,121 @@
 Manifests
 =========
 
-Bla bla XML upstream, revision, where to put files and so on.
+Repo [#repotool]_ is a tool for using several git repositories in the same
+build. It features a `manifest` format using XML where one defines the git
+repositories and revisions one wants to use. It is possible to include manifests
+in other manifests.
 
-Structure, include other manifests etc.
+Features 101
+------------
+* Download several git repos simultaneously.
+* Use different revisions for different repos.
+    * Revisions can be branches, tags or commit hashes.
+* Combine manifests together.
+    * Create more specialized manifests from a common base.
+* Default settings for revision etc.
+* Separate remotes and actual repos.
+
+Usage in yocto
+--------------
+When doing yocto builds, the repo tool is useful to have the various yocto
+layers synced towards the same yocto release (typically, a poky release).
+
+Revisions in repo manifests can be branch names, tags, or commit hashes. For
+maximum reproducability, commit hashes is a good solution, but in order to be
+able to follow layers as they move, branch names are suitable.
+
+If one wants to support several yocto releases, it is possible to use different
+branches in the manifest repo, where the manifest in each branch would then
+point to the corresponding branch (or a commit on that branch) in the layers
+used.
+
+Example
+^^^^^^^
+Below is an example of a manifest from PELUX [#pelux-manifests]_ showcasing the
+format.
+
+.. code-block:: xml
```

Comment:
I think I'd prefare `.. literalinclude:: filename` instead. In theory one could use the pelux-manifests repo as a submodule and use the actuall file, but it might be too much hassle.

Reply:
Good point. But on that topic: do we want the FULL manifest? I was considering shortening it a bit actually for brevity.

Reply:
I wouldn't mind if it was shorter.

Reply:
If we could very easily use the real file without maintenance overhead, I'd vote for a shortened example designed to not be maintenance heavy.